packages/e/eabase/xmake.lua

```diff
@@ -0,0 +1,22 @@
+package("eabase")
+
+    set_homepage("https://github.com/electronicarts/EABase")
+    set_description("EABase is a small set of header files that define platform-independent data types and platform feature macros.")
+    set_license("BSD-3-Clause")
+
+    add_urls("https://github.com/electronicarts/EABase/archive/$(version).tar.gz")
+    add_versions("2.09.06", "981f922441617152b841585c0fc0bd205bd898c758016fa4985599a63c5a6e16")
+
+    add_deps("cmake")
+
+    on_install("windows", "linux", "macosx", function (package)
+        import("package.tools.cmake").install(package)
```

Comment:
ci fails, includedirs not found. 

Maybe this project does not have a built-in cmake_install script, so you may need to use os.cp to install the header files to the include directory. then use cmake.build like the previous package.

```
finding eabase from xmake ..
{ 
  license = "BSD-3-Clause", 
}
```